package.yaml

```diff
@@ -25,6 +25,7 @@ dependencies:
 - base >= 4.8 && < 5
 - text < 2
 - bytestring >= 0.9 && < 0.12
+- base16-bytestring >= 1.0 && < 1.1
```

Comment:
Is there any particular reason for the lower bound here? The first version of `base16-bytestring` on Hackage (`0.1`) has the `encode` function in `Data.ByteString.Base16` already. As far as I can tell from their changelog, there is no reason not to support a lower bound, particularly since it would avoid the `extra-deps`?

Reply:
It's just that it's a safer choice to limit to one major release, but I have no strong opinion (relaxed the lower bound to 0.1)

Reply:
That's fair, probably safe in this case though!